src/lib/infractionEmbedPagination.ts

```diff
@@ -43,7 +43,7 @@ export async function createPaginatedInfractionEmbedHandler(initialInteraction:
     const message = await initialInteraction.followUp({ embeds: [embeds[currentPage]], components: [buttonRow1, buttonRow2] });
 
     const filter = (interaction: Interaction) => interaction.user.id === user;
-    const collector = message.createMessageComponentCollector({ filter, time: 120000 });
+    const collector = message.createMessageComponentCollector({ filter, time: 5_000 });
```

Comment:
```suggestion
    const collector = message.createMessageComponentCollector({ filter, time: 120_000 });
```

I assume this was changed for testing?